salt/orch/update-etc-hosts.sls

```diff
@@ -1,4 +1,4 @@
-{%- set updates_all_target = 'P@roles:(admin|kube-(master|minion)) and G@bootstrap_complete:true and not P@.*_in_progress:true' %}
+{%- set updates_all_target = 'P@roles:(admin|kube-(master|minion)) and G@bootstrap_complete:true and not G@update_in_progress:true and not G@removal_in_progress:true' %}
```

Comment:
Missing bootstrap_in_progress:true

Reply:
Oh I skipped it because of `bootstrap_complete:true` but it won't work when re-running the orchestration because of a settings change for example... Fixing.